source/security/index.markdown

```diff
@@ -1,11 +1,81 @@
 ---
 title: "Security"
-description: "Information about disclosing security vulnerabilities in Home Assistant."
+description: "Home Assistant takes its security seriously. This page contains information about how we handle security issues, how to report them, and information on past security issues."
 ---
 
-If you think that you have found a security vulnerability in Home Assistant, please disclose it to us via our security e-mail address at [security@home-assistant.io](mailto:security@home-assistant.io).
+Home Assistant takes its security seriously. We will do everything in our power to ensure that our users are safe.
 
-Please do not make vulnerabilities public without notifying us and giving us at least 3 days to respond.
+This page is intended to provide information about how we handle security issues and information about security issues we have [handled in the past](#past-advisories).
```

Comment:
"handle issues that are reported to us" or "handle issues whenthey are reported to us"; can't handle those we don't know about